src/etl/ETLHelpers.h

```diff
@@ -103,17 +115,26 @@ class ThreadSafeQueue
     std::optional<uint32_t> maxSize_;
 
 public:
-    /// @param maxSize maximum size of the queue. Calls that would cause the
-    /// queue to exceed this size will block until free space is available
+    /**
+     * @brief Create an instance of the queue
+     *
+     * @param maxSize maximum size of the queue. Calls that would cause the queue to exceed this size will block until
+     * free space is available
+     */
     ThreadSafeQueue(uint32_t maxSize) : maxSize_(maxSize)
```

Comment:
It is not the issue of this PR, just found the maxSize_ can not be false, but member functions keep checking it.

Reply:
Well, it can be if the default constructor is used. However, in this codebase we do not use the default constructor so it could be simplified to just `uint32_t maxSize_` i think.